### PR TITLE
implemented ellipse component, #8

### PR DESCRIPTION
**Pull request by HVish**

### CODE REVIEW

File: src/components/ellipse/Ellipse.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+} from '../../styleTransformers/transformLayoutStyleProperties';
+import { useYogaLayout } from '../../hooks/useYogaLayout';
+import { transformBlendProperties, BlendStyleProperties } from '../../styleTransformers/transformBlendProperties';
+import {
+    GeometryStyleProperties,
+    transformGeometryStyleProperties
+} from '../../styleTransformers/transformGeometryStyleProperties';
+import { useFillsPreprocessor } from '../../hooks/useFillsPreprocessor';
+
+export interface EllipseNodeProps extends DefaultShapeProps, CornerProps {
```
Comment (LosYear) [inline src/components/ellipse/Ellipse.tsx]: I suggest calling it `EllipseProps` like any other interfaces (e.g. `TextProps`, `GroupProps`, etc)

Comment (HVish) [reply src/components/ellipse/Ellipse.tsx]: Sure. I agree with you.

**Comment by ilyalesik** [comment]
Hi, @HVish! Thank you for the pull request.
May I ask you to add [arcData](https://www.figma.com/plugin-docs/api/ArcData/) prop support at Ellipse? Figma API has it:
```typescript
interface EllipseNode extends DefaultShapeMixin, ConstraintMixin, CornerMixin {
  readonly type: "ELLIPSE"
  clone(): EllipseNode
  arcData: ArcData
}
```

**Comment by HVish** [comment]
@ilyalesik Sure! I wasn't sure if arcData should be added to props since it was not mentioned in [this](https://www.figma.com/plugin-docs/api/EllipseNode/) documentation.

**Comment by ilyalesik** [comment]
Great work! Thanks!

**Pull request merged**